src/components/modules/toolbar/blockSettings.ts

```diff
@@ -130,6 +141,15 @@ export default class BlockSettings extends Module {
   public close(): void {
     this.nodes.wrapper.classList.remove(this.CSS.wrapperOpened);
 
+    /**
+     * Restore selection to Block
```

Comment:
need to describe current condition: why we need to check if the selection is at editor